packages/admin/src/cli.ts

```diff
@@ -1,168 +1,203 @@
 import * as fs from 'fs';
 import * as yargs from 'yargs';
 import * as evm from './evm';
-import * as contract from '.';
+import * as admin from '.';
+import { exit } from 'process';
+
+const COMMON_COMMAND_ARGUMENTS = {
+  airnodeRrpCommands: {
+    providerUrl: {
+      type: 'string',
+      demandOption: true,
+      describe: 'URL of the Ethereum provider',
+    },
+    airnodeRrp: {
+      type: 'string',
+      describe: 'Address of the deployed AirnodeRrp contract',
+    },
+  },
+  mnemonicCommands: {
+    mnemonic: {
+      type: 'string',
+      demandOption: true,
+      describe: 'Mnemonic of the wallet',
+    },
+    derivationPath: {
+      type: 'string',
+      describe: 'Derivation path to be used for deriving the wallet account',
+    },
+  },
+  requesterAdmin: {
+    type: 'string',
+    demandOption: true,
+    describe: 'Address of the requester admin',
+  },
+  requesterIndex: {
+    type: 'string',
+    demandOption: true,
+    describe: 'Requester index',
+  },
+  airnodeId: {
+    type: 'string',
+    demandOption: true,
+    describe: 'Airnode id',
```

Comment:
Yeah, I didn't pay much focus into descriptions as they should.

It would also be nice to offer more examples using http://yargs.js.org/docs/#api-reference-examplecmd1-desc1-cmd2-desc2

Reply:
Actually let me track it under https://github.com/api3dao/airnode/issues/286